vic/drivers/cesm/bld/build_vic_namelist

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env python
+from __future__ import print_function
+import os
+import stat
+import shutil
+try:
+    from ConfigParser import SafeConfigParser
+except ImportError:
+    from configparser import SafeConfigParser
```

Comment:
no error necessary, I'll add a comment but this just handles the python 2 to 3 conversion of `ConfigParser` to `configparser`.